### ISSUE TITLE
Asses the accessibility of the Modal component

### ISSUE THREAD

**Issue by tobiasljungstrom** [SKF/ferris#255]
Asses the accessibility of the Modal component and mark it in the component table.

The component should meet the following criteria (if applicable):

- [ ] Tab-able with a clear focus state
- [ ] Screen reader should be able to traverse it
- [ ] Custom control should work with screen reader
- [ ] Focus should be directed to any new items that pop up on the screen
- [ ] Color and contrast should be good
